tests/unit/models/base-channel-test.js

```diff
@@ -70,6 +70,28 @@ module('Unit | Model | base-channel', function(hooks) {
     assert.equal(channel.get('sortedMessages').length, 0);
   });
 
+  //
+  // sortedUserList
+  //
+
+  test('#sortedUserList returns an empty array if there are no users', function(assert) {
+    const channel = BaseChannel.create();
+
+    assert.equal(typeof channel.get('sortedUserList'), 'object');
+    assert.equal(channel.get('sortedUserList').length, 0);
+  });
+
+  test('#sortedUserList returns a list of sorted usernames', function(assert) {
+    const channel = BaseChannel.create();
+
+    channel.set('userList', ['silverbucket', 'XioNox', 'raucau', '@operator',
+                             'gregkare', 'galfert', '@MrOps']);
```

Comment:
I think we might also want to test other prefixes that are commonly used in IRC: http://www.geekshed.net/2009/10/nick-prefixes-explained/